<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MWin_BaseWin_AWM</class>
 <widget class="QMainWindow" name="MWin_BaseWin_AWM">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1900</width>
    <height>950</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="CWin_BaseWin_AWM">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QStackedWidget" name="BseM_Stack">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="NEX_Stack1">
       <layout class="QGridLayout" name="gridLayout_2"/>
      </widget>
      <widget class="QWidget" name="NEX_Stack2">
       <layout class="QGridLayout" name="gridLayout_3"/>
      </widget>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QFrame" name="NEX_StatusBar">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>30</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Courier New</family>
       </font>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QGridLayout" name="gridLayout_7">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item row="0" column="2">
        <widget class="QLabel" name="OQL_ProcessStatus">
         <property name="minimumSize">
          <size>
           <width>600</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>600</width>
           <height>30</height>
          </size>
         </property>
         <property name="text">
          <string>Process Status </string>
         </property>
        </widget>
       </item>
       <item row="0" column="5">
        <widget class="QProgressBar" name="OQL_ProgressBar">
         <property name="minimumSize">
          <size>
           <width>300</width>
           <height>20</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>200</width>
           <height>20</height>
          </size>
         </property>
         <property name="value">
          <number>24</number>
         </property>
        </widget>
       </item>
       <item row="0" column="12" alignment="Qt::AlignRight">
        <widget class="QLabel" name="OQL_ForQuery">
         <property name="minimumSize">
          <size>
           <width>250</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>For any Query +91 99525 09481</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0" alignment="Qt::AlignLeft">
        <widget class="QLabel" name="OQL_Emoji">
         <property name="minimumSize">
          <size>
           <width>50</width>
           <height>30</height>
          </size>
         </property>
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Emoji&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item row="0" column="6">
        <spacer name="NEX_horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>200</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="1">
        <widget class="Line" name="NEX_line">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
       </item>
       <item row="0" column="11">
        <widget class="QLabel" name="OQL_CurrentDate">
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>30</height>
          </size>
         </property>
         <property name="text">
          <string>DD/MM/YYYY</string>
         </property>
        </widget>
       </item>
       <item row="0" column="8">
        <spacer name="NEX_horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>200</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="7">
        <widget class="QLabel" name="OQL_SoftwareName">
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>Twink_AWM</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1900</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Courier New</family>
     <pointsize>10</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <widget class="QMenu" name="menuNavigate">
    <property name="font">
     <font>
      <family>Courier New</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Navigate</string>
    </property>
    <addaction name="actionHome"/>
    <addaction name="actionRegister"/>
    <addaction name="actionMater_User"/>
   </widget>
   <widget class="QMenu" name="menuAttendance">
    <property name="font">
     <font>
      <family>Courier New</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Attendance</string>
    </property>
    <addaction name="actionCreate"/>
    <addaction name="actionView"/>
    <addaction name="actionPunchTrack"/>
    <addaction name="actionPersPunchTrack"/>
    <addaction name="actionAdvance"/>
   </widget>
   <widget class="QMenu" name="menuReports">
    <property name="font">
     <font>
      <family>Courier New</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Reports</string>
    </property>
    <addaction name="actionWage_Monthly_Calculation"/>
    <addaction name="actionWage_Category_Split"/>
   </widget>
   <addaction name="menuNavigate"/>
   <addaction name="menuAttendance"/>
   <addaction name="menuReports"/>
  </widget>
  <action name="actionHome">
   <property name="text">
    <string>Home</string>
   </property>
  </action>
  <action name="actionRegister">
   <property name="text">
    <string>Register</string>
   </property>
  </action>
  <action name="actionMater_User">
   <property name="text">
    <string>Mater User</string>
   </property>
  </action>
  <action name="actionCreate">
   <property name="text">
    <string>Create</string>
   </property>
  </action>
  <action name="actionView">
   <property name="text">
    <string>View</string>
   </property>
  </action>
  <action name="actionPunchTrack">
   <property name="text">
    <string>PunchTrack</string>
   </property>
  </action>
  <action name="actionPersPunchTrack">
   <property name="text">
    <string>(P) Punch Track </string>
   </property>
  </action>
  <action name="actionAdvance">
   <property name="text">
    <string>Advance</string>
   </property>
  </action>
  <action name="actionWage_Monthly_Calculation">
   <property name="text">
    <string>Wage Monthly Calculation</string>
   </property>
  </action>
  <action name="actionWage_Category_Split">
   <property name="text">
    <string>Wage Category Split</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
